test/integration/controlling_test.exs

```diff
@@ -12,33 +12,33 @@ defmodule Oban.Integration.ControllingTest do
     end
 
     test "starting individual queues dynamically" do
-      start_supervised_oban!(queues: [alpha: 10])
+      name = start_supervised_oban!(queues: [alpha: 10]).name
```

Comment:
The look of this may take me some getting used to. The fact that it is a unique reference for every test is fantastic though.

Reply:
Hopefully it's a bit simpler now, after I applied your proposal of only returning the name.